builders/build/templates/evm.md

```diff
@@ -33,7 +33,7 @@ Besides the modules and configurations that make the Tanssi EVM appchain templat
 
 To reach full Ethereum compatibility, these specific modules are also included:
 
-- **[pallet_evm](https://docs.rs/pallet-evm/latest/pallet_evm){target=\_blank}** - the EVM pallet allows for unmodified EVM bytecode to be executed in a Substrate-based blockchain. It uses the Rust-based [SputnikVM](https://github.com/rust-ethereum/evm){target=\_blank} as the underlying EVM engine
-- **[pallet_ethereum](https://docs.rs/pallet-ethereum/latest/pallet_ethereum){target=\_blank}** - the Ethereum pallet works together with the EVM pallet to provide full emulation for Ethereum block processing. Among many other tasks, it is responsible for creating emulated Ethereum blocks for Ethereum-specific components such as EVM logs
+- **[EVM module](https://docs.rs/pallet-evm/latest/pallet_evm){target=\_blank}** - it adds support for unmodified EVM bytecode execution on a Tanssi appchain. It uses the Rust-based [SputnikVM](https://github.com/rust-ethereum/evm){target=\_blank} as the underlying EVM engine
```

Comment:
Same comment as before, if we are using modules at the top we can remove them here?